db/migrate/20220429092052_create_order_cycles_shipping_methods.rb

```diff
@@ -1,17 +1,16 @@
 class CreateOrderCycleShippingMethods < ActiveRecord::Migration[6.1]
   def up
-    create_table :order_cycle_shipping_methods do |t|
+    create_table :order_cycles_shipping_methods, id: false do |t|
       t.references :order_cycle
       t.references :shipping_method, foreign_key: { to_table: :spree_shipping_methods }
```

Comment:
The official documentation recommends this:
```
t.belongs_to :order_cycle
t.belongs_to :shipping_method
```

Would that be a better option here?

Reply:
Yes that's a bit clearer, fixed by https://github.com/openfoodfoundation/openfoodnetwork/pull/9262/commits/19e4e6e46c8bdfacdad2fefb67d392b585f35eac